timeit/src/Pages/TaskPage/Taskpage.js

```diff
@@ -44,12 +74,26 @@ const Taskpage = () => {
           </button>
         </div>
         {state.tasks.map((ele) => (
-          <div key={ele} className="task-details">
-            <p>{ele}</p>
+          <div key={ele.taskName} className="task-details">
+            <p>{ele.taskName}</p>
             <div className="task-buttons">
-              <button>Timer</button>
+              <button
+                onClick={() => {
+                  console.log(ele);
```

Comment:
suggestion: move this onclick code to separate function and call that function here. It makes code more readable